Appraisals

```diff
@@ -949,6 +949,7 @@ elsif ruby_version?('2.5')
     gem 'shoryuken'
     gem 'sidekiq'
     gem 'sneakers', '>= 2.12.0'
+    gem 'bunny', '~> 2.19.0' # uninitialized constant OpenSSL::SSL::TLS1_3_VERSION for jruby, https://github.com/ruby-amqp/bunny/issues/645
```

Comment:
Interesting. This updated Ruby 2.5.9 as well since we only have one section for 2.5, which covers both Ruby and JRuby. I wonder if we should split Ruby and JRuby at some point.

In fact I've been wondering for a while if we should extract all these per-version appraisal sections into their own files, something like:

```
# in Appraisals
eval(File.read("appraisals/#{RUBY_ENGINE}_#{RUBY_ENGINE_VERSION.gsub(/\.\d+$/, '')}.appraisals"))
```

This would mirror the prefixes we have in `gemfiles`, the ruby image names, the docker-compose ruby services...

Reply:
I like the idea of seperation. Currently with this setup, any changes with 2.5 will have to run

```
bundle exec rake appraisal:install[jruby-9.2.8.0,jruby-9.2,2.5]
```